recipes/ncl/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+export CC=${PREFIX}/bin/gcc
+export CXXFLAGS="-fPIC"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+export CFLAGS="-I${PREFIX}/include"
```

Comment:
If possible try to do this like: `export CFLAGS="-I${PREFIX}/include $CFLAGS"` so the pre set flags are preserved.